.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,8 +7,5 @@ Describe the changes made in your pull request here.
 Ensure you completed **all of the steps** below before submitting your pull request:
 
 - [ ] Added natspec comments?
-- [ ] Ran `forge snapshot`?
-- [ ] Ran `pnpm gas-report`?
-- [ ] Ran `pnpm lint`?
-- [ ] Ran `forge test`?
+- [ ] Ran `pnpm adorno`?
```

Comment:
There could be an argument that removing `forge test` here is not ideal, but on the other hand, this will then latest fail during CI when there's failing tests. So I think it's fine, but I still wanted to point it out.

Reply:
forge test is implicit on forge snapshot and pnpm gas-report.